### ISSUE TITLE
Arduino IDE builds are broken

### ISSUE THREAD

**Issue by ZZ-Cat** [ZZ-Cat/CRSFforArduino#93]
### Discussed in https://github.com/ZZ-Cat/CRSFforArduino/discussions/92

<div type='discussions-op-text'>

<sup>Originally posted by **ZZ-Cat** March  5, 2024</sup>
This means that if you try to build CRSF for Arduino v1.0.0 in the Arduino IDE, the build process will fail with the error  
`Compilation error: CRSFforArduino.hpp: No such file or directory.`

This appears to be caused by a line break in one of the keys in CRSF for Arduino's `library.properties` file.  
The key in question is the `paragraph` key, which is suspected to be the likely culprit.  

I am currently investigating the cause of this bug, and if the above is the cause, then it's a fairly quick fix.  
However, in my own experiences in coding, _nothing_ is as straightforward (or as quick) as it may seem at face value. Once the fix is done, a cut from the [Version 1.0.x Maintenance Branch](../tree/1.0.x-Maintenance-Branch) will be available in the [Releases](../releases) section as Version 1.0.1.</div>

---

This has been created from the PSA above, to get development of this bug into the pipeline.  
Currently, this is top priority, thus delaying development of Version 1.1.0 until this is resolved.
